docs/content/4.sdk/2.getting-started/4.logger.md

```diff
@@ -0,0 +1,168 @@
+# Logger
+
+The Logger provides a logging utility for Alokai Storefront projects. It allows you to log messages at various levels and ability to provide additional context. 
+
+It is created to provide a unified way of logging messages across the project along with providing additional data out of the box.
+
+## Installation
+
+If the Logger is already included in the Alokai Storefront, you can skip this step.
```

Comment:
If it is not, then what? In which circumstances it might be missing?

Reply:
It might be the case that a client already has a delivered storefront that doesn't yet have logger implemented. For such case it has to be installed, otherwise this step is not required.